tests/Operation/WatchFunctionalTest.php

```diff
@@ -749,9 +745,10 @@ public function testNonResumableErrorCodes($errorCode)
     public function provideNonResumableErrorCodes()
     {
         return [
-            [136], // CappedPositionLost
-            [237], // CursorKilled
-            [11601], // Interrupted
+            'CappedPositionLost' => [136],
+            'CursorKilled' => [237],
+            'Interrupted' => [11601],
+            'ChangeStreamFatalError' => [280],
```

Comment:
Reminder to remove this once https://github.com/mongodb/mongo-php-driver/pull/1018 is merged and [PHPC-1419](https://jira.mongodb.org/browse/PHPC-1419) is released.

Reply:
Removed along with the corresponding functionality in `ChangeStream` now that we're using RC-1.